tests/lxmert/test_modeling_tf_lxmert.py

```diff
@@ -486,135 +488,31 @@ def check_hidden_states_output(config, inputs_dict, model_class):
             config.output_hidden_states = True
             check_hidden_states_output(config, inputs_dict, model_class)
 
-    def test_pt_tf_model_equivalence(self):
-        from transformers import is_torch_available
-
-        if not is_torch_available():
-            return
-
+    def prepare_pt_inputs_from_tf_inputs(self, tf_inputs_dict):
         import torch
```

Comment:
Can I add `import torch` here without `is_torch_available` or `require_torch`? This method will be called only inside `test_pt_tf_model_equivalence`, which is already decorated with `is_pt_tf_cross_test`.

Reply:
That's just a marker that reads an env variable, so I think it should have the `require_torch` just in case, but I'm not sure if we are very consistent with that. @LysandreJik might know better.

Reply:
I don't think it really matters as it is indeed already decorated with the `is_pt_tf_cross_Test`. We don't have a convention set, so feel free to choose the simplest approach.